tools/development/helpers/docs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+project_directory="$(git rev-parse --show-toplevel)"
+project_name_camel_case=$(echo "$project_directory" | cut -d '-' -f4- | sed -r 's/(^|-)([a-z])/\U\2/g')
+docs_directory="${project_directory}/docs"
+
+pushd "${docs_directory}" > /dev/null
+
+pip install -r requirements.txt
```

Comment:
Will this look for a `requirement.txt` in the docs directory? I don't see it there in any of the repos. (might be misunderstanding what's happening here.

Reply:
Yes that's correct, there's a `requirement.txt` in the OSTk Astro repo. I will update the other repos soon with the required files to generate docs.